### DISCUSSION TITLE
direct access to Screenshot

### DISCUSSION THREAD

**Discussion by MicheleVilla** [EdgeTX/edgetx#3994] [Ideas]
it could be very useful to have direct access to capture a screenshot without necessarily having to implement a special function
for example in B & W radios by long pressing one of the keys or a combination of them
I wouldn't know how to do this on color radios

**Comment by raphaelcoeffic** [comment]
A combination would be good, however we don't support that yet. Color radios also just use special functions to that purpose.

**Comment by MicheleVilla** [comment]
the sentence I used is incorrect
_"without necessarily having to **USE** a special function"_
the special screenshot function is also already present in B & W radios and is very useful.
A further possibility would also be to use a simple key combination instead of having to program it using the special functions associated with a switch on any page you are on
In this way the capture of a screenshot is more immediate and does without a switch
